source/basic.tex

```diff
@@ -3989,15 +3989,15 @@
 \rSec3[basic.stc.dynamic]{Dynamic storage duration}%
 
 \rSec4[basic.stc.dynamic.general]{General}%
-\indextext{storage duration!dynamic|(}
 
 \pnum
-Objects can be created dynamically during program
-execution\iref{intro.execution}, using
+Object with \defnadj{dynamic}{storage duration} are created using
```

Comment:
Can we reverse the phrasing here?  The storage duration is a consequence of the syntax used to create the object, not the other way round: "Objects created using X and Y have dynamic storage duration."  That seems to be the way we phrase it for the other storage durations (never mind object vs. variable).

Reply:
Done. The only reasonable way to accommodate the note on *delete-expression*s was to pull it into a completely separate sentence though, which is why I originally refrained from doing that.

I think it's overall for the better now though; good suggestion.